internal/conf/computed.go

```diff
@@ -296,6 +296,12 @@ func IsBuiltinSignupAllowed() bool {
 	return false
 }
 
+// IsAccessRequestsEnabled returns whether request access experimental feature is enabled or not.
+func IsAccessRequestsEnabled() bool {
+	experimentalFeatures := Get().ExperimentalFeatures
+	return experimentalFeatures == nil || experimentalFeatures.AccessRequestsEnabled == nil || *experimentalFeatures.AccessRequestsEnabled
```

Comment:
Shouldn't it be other way around?
```suggestion
	return experimentalFeatures != nil && experimentalFeatures.AccessRequestsEnabled != nil && *experimentalFeatures.AccessRequestsEnabled
```

Reply:
I intentionally used this approach because we want an experimental feature enabled by default, even if not set.